src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -2370,6 +2372,32 @@ public void testBuild() {
         assertEquals(BuildStatus.BUILT_INVALID, networkModificationTreeService.getBuildStatus(modificationNode5.getId()));
     }
 
+    @Test
+    public void deleteModificationRequest() {
+        createStudy("userId", CASE_UUID, false);
+        UUID studyNameUserIdUuid = studyRepository.findAll().get(0).getId();
```

Comment:
`studyNameUserIdUuid` is a strange name, is `studyUuid` not sufficient ?

Reply:
done